frontend/src/components/styles/SignUp.css

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;500;600;700&display=swap');
+
+:root{
+    --red-bg-color: #ff7a7a;
```

Comment:
Looks like you copied code from somewhere?

Reply:
yeah, any issues in that as font suits this component

Reply:
Not an issue with that. But please refine the code. 
Don't use properties on :root. Reduce redundant properties.

Reply:
@ashishnagpal2498 : Done, Please check now.

Reply:
![InkedsignupComp_LI](https://user-images.githubusercontent.com/36301481/107883527-8ae21200-6f15-11eb-8936-933f4d96404f.jpg)
Why do you need this when there's already a button.
Also, instead of state make a route for " SignUp ".

Reply:
@ashishnagpal2498 : Done, Please have a look - I haven't used existing icon as it was mapped to login - now updated to Signup